src/ipc/Kids.cc

```diff
@@ -29,25 +29,16 @@ void Kids::init()
 
     storage.reserve(NumberOfKids());
 
-    char kid_name[32];
-
-    // add Kid records for all workers
-    for (int i = 0; i < Config.workers; ++i) {
-        snprintf(kid_name, sizeof(kid_name), "(squid-%d)", (int)(storage.size()+1));
-        storage.push_back(Kid(kid_name));
-    }
+    for (int i = 0; i < Config.workers; ++i)
+        storage.push_back(Kid("squid", storage.size() + 1));
```

Comment:
please prefer emplace methods over push especially when the parameter is newly constructed like this. It allows the compiler to choose move over a double-copy here.

same for the others in this method.

Reply:
Done.